Base/res/themes/Default.ini

```diff
@@ -74,8 +74,11 @@ TrayText=#ffffff
 
 [Flags]
 IsDark=false
+IsTitleCenter=false
+HasRoundedBorders=false
```

Comment:
These don't appear to have defaults in the code, so they should be present in all other themes as well.

Reply:
> These don't appear to have defaults in the code, so they should be present in all other themes as well.

Flags default to false if undefined, just added those properties to the default theme for extra verboseness.

Reply:
The better approach is to just check if `BorderRadius < 0` though. Will fix that.